.github/workflows/claude_runner.yml

```
@@ -14,7 +14,7 @@ concurrency:
 jobs:
   get_issue_and_assign:
     runs-on: ubuntu-latest
-    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)